android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -49,6 +51,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
 
+        downloadCompleteHandler = new DownloadCompleteHandler(getContext(), R.string.menu_feed);
```

Comment:
Yes, because both classes are completely different. One listens to Event Info download, and your use case is just showing an intermediate progress dialog I guess?